rpcd/etc/openstack_deploy/user_extras_variables.yml

```diff
@@ -95,18 +95,32 @@ maas_filesystem_critical_threshold: 90.0
 #   - { package: "rabbitmq-server", origin: "www.rabbitmq.com" }
 #   - { package: "*", release: "MariaDB" }
 
-# Ceph
+# Ceph options
+# fsid is the unique identifier for your object store.
 fsid: '{{ fsid_uuid }}'
+# directory for backing up ceph keys.
 fetch_directory: /etc/openstack_deploy/ceph_fetch
+# Use stable version of ceph
 ceph_stable: true
+# Specify ceph release name
 ceph_stable_release: hammer
+# Enablie openstrack support inside the ceph-ansible playbooks
 openstack_config: true
+# Use raw journal devices
 raw_multi_journal: true
+# Set the journal size to: "Size of journal device / number of devices for which it is a journal"
+# E.g. Given a 400G journal disk with 4 disks using it as their journal device, the journal size should be 100G each or 100000
```

Comment:
"Given a 400G journal disk with 5 disks using it as their journal device, the journal size should be 80G each or 80000"

May as well use the ratio that we'll have in prod.